virtcontainers/qemu.go

```diff
@@ -721,6 +723,11 @@ func (q *qemu) startSandbox(timeout int) error {
 	if err != nil {
 		return err
 	}
+	// append logfile only on debug
+	if q.config.Debug {
+		q.qemuConfig.LogFile = filepath.Join(vmPath, "qemu.log")
```

Comment:
If we create the log file in the vmPath and if the QEMU crashes for some reason, wouldn't the logs also get removed as part of the sandbox cleanup? Would it help if we can leave it elsewhere? (I understand that poses an issue if orphaned files on the host created by Kata, but would help for debug later wouldn't it?)

Reply:
Good point! Another option is to read the log file and output it to the logger system.

Reply:
+1 for logging to the logger system

Reply:
That would work for sure.

Reply:
I've added code to print it to the logger system when debug is on. I'm not sure if it useful to give users options to configure the log file location though,  because this is only for debug.